### ISSUE TITLE
Trouble installing package in fresh environment

### ISSUE THREAD

**Issue by mrland99** [MarioniLab/DropletUtils#90]
I'm having trouble installing DropletUtils in both new conda and docker environments using `BiocManager::install("DropletUtils")`.

I've double-checked that my R version is compatible (4.0+) as well as BiocManager. 

Sample dockerfile code (that I essentially also replicate in conda):
1) install r-base
2) install BiocManager
3) install DropletUtils

```
#Install R
RUN apt install -y dirmngr apt-transport-https ca-certificates software-properties-common gnupg2
RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-key '95C0FAF38DB3CCAD0C080A7BDC78B2DDEABC47B7'
RUN add-apt-repository 'deb https://cloud.r-project.org/bin/linux/debian buster-cran40/'
RUN apt update
RUN apt install -y r-base build-essential


# Install packages
RUN R -e "install.packages('BiocManager')"
RUN R -e "BiocManager::install('DropletUtils', dependencies = TRUE); if (!library(DropletUtils, logical.return=T)) quit(status=10)"
```
I'm wondering if this is a common problem anyone else has had? Are there other dependencies that DropletUtils depends on that I'm missing?

TLDR: Anyone successfully installed DropletUtils from a new conda env?

**Comment by mrland99** [comment]
For what it's worth I have successfully installed it on my local computer via RStudio, but I am unable to replicate it in a new conda environment.

**Comment by jonathangriffiths** [comment]
Please could you include the error message you are seeing when actually installing the package?

The build system is okay so I think it's something specific on your end. If you're happy with docker, you might consider using the [bioconductor docker image](https://www.bioconductor.org/help/docker/) as this contains all the dependencies required for bioconductor packages.

**Comment by jonathangriffiths** [comment]
Closing for inactivity, feel free to come back with more questions if this issue returns.
